Apps/Admin/Server/Services/DashboardService.cs

```diff
@@ -95,12 +97,13 @@ public int GetRecurrentUserCount(int dayCount, string startPeriod, string endPer
         }
 
         /// <inheritdoc />
-        public RequestResult<IEnumerable<Database.Models.MessagingVerification>> GetMessageVerifications(UserQueryType queryType, string queryString)
+        public RequestResult<IList<MessagingVerificationModel>> GetMessageVerifications(UserQueryType queryType, string queryString)
```

Comment:
I think we should return IEnumerable unless we have a specific reason to need a list.

The comment I left on AdminWebClient's DashboardService applies here as well for populating the payload.

The initial value of ResourcePayload can be set to `Enumerable.Empty<MessagingVerificationModel>()` instead of an empty list.